packages/core/src/StateNode.ts

```diff
@@ -278,6 +278,7 @@ class StateNode<
   };
 
   private idMap: Record<string, StateNode<TContext, any, TEvent>> = {};
+  public tags?: string[];
```

Comment:
Usually, we accept `SingleOrArray<T>` for this kind of stuff - should we keep this consistent for this?

Reply:
Yeah, that's a good idea

Reply:
Done + test added